csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -71,21 +85,26 @@ public sealed class OutgoingRequestFrame : OutgoingFrame
         /// instances.</param>
         /// <param name="context">An optional explicit context. When non null, it overrides both the context of the
         /// proxy and the communicator's current context (if any).</param>
+        /// <param name="cancel">A cancellation token that receives the cancellation requests.</param>
         /// <param name="args">The argument(s) to write into the frame.</param>
         /// <param name="writer">The <see cref="OutputStreamWriter{T}"/> that writes the arguments into the frame.
         /// </param>
         /// <returns>A new OutgoingRequestFrame.</returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1068:CancellationToken parameters must come last",
+            Justification = "Delegates and lambda function should be the pass as the last argument")]
```

Comment:
Fine, let's put the cancellation token last.

Reply:
moved the token to the end